packages/utilities/adaptive-ui/src/design-tokens/color.ts

```diff
@@ -63,11 +64,8 @@ export const accentFillFocusDelta = createNonCss<number>(
     "accent-fill-focus-delta"
 ).withDefault(0);
 
-/** @public */
-export const accentFillRecipe = createNonCss<InteractiveColorRecipe>(
-    "accent-fill-recipe"
-).withDefault({
-    evaluate: (element: HTMLElement, reference?: Swatch): InteractiveSwatchSet =>
+const accentFill = createInteractiveColorTokens(
```

Comment:
How does having all of this in a set affect tree shaking? Are the cases where someone would want to import only `accentFillRest` and expect the other accent interactive tokens to be tree shaken away, or is the assumption that if you import one, then you'll need to import the rest in order to have a consistent look across element states?

Reply:
My thought toward addressing this is that all instances of creating those individual tokens has been moved into a single instance of the function above. Yes, this means that if you use _any_ `interactiveColorTokens` you get the code for `rest`, `hover`, etc. That code above could probably be consolidated as well.
At least if you use even _two_ `interactiveColorTokens` you get to share the code to create those stateful tokens.
I think that actually would make code size smaller, but I haven't tested it.

Reply:
Actually, `rest` etc. above could even be functions that cache the output. Then if you never use a `hover` state or whatever, you never `create` the `DesignToken`. That seems efficient from the perspective of the amount of code running compared to what is used.